src/Mvc/Mvc.ViewFeatures/test/Rendering/SystemTextJsonHelperTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
```

Comment:
extra line. Other people tell me it's important not to have these.

Reply:
🙏